app.py

```diff
@@ -44,6 +44,10 @@ def index():
 				TRAVIS_TAG = urlify(TRAVIS_TAG)#this will fix url issue
 				os.environ["TRAVIS_TAG"] = TRAVIS_TAG
 				os.environ["event_url"] = event_url
+				file = open('travis_script_1.sh', 'r')
```

Comment:
I recommend to use a less generic name for the file handle. But that is just style.